packaging/kibana.spec

```diff
@@ -0,0 +1,55 @@
+Summary:       custom logrhythm kibana
+Name:          kibana
+Version:       %{version}
+Release:       1%{?dist}
+License:       https://github.com/joyent/node/blob/master/LICENSE
+Group:         Development/Tools
+URL:           https://github.com/joyent/%{name}
+Source:        http://nodejs.org/dist/v%{version}/%{name}-v{%version}.tar.gz
+
+%description
+kibana build for logrhythm
+
+%prep
+#cleanup
+cd %_builddir
+rm -rf %{name}
+mkdir %{name}
+cd %{name}
+#extract sources
+tar xzf %_sourcedir/%{name}-%{version}.tar.gz
+if [ $? -ne 0 ]; then
+   exit $?
+fi
+#modify shasum to use sha1sum for centos
+sed -i s/\'shasum/\'sha1sum/g tasks/create_shasums.js
+sed -i s/0.10.x/0.10.42/g .node-version
+#ensure nvm is installed
+NVM_DIR="%_builddir/%{name}/nvm" bash scripts/install_nvm.sh
+source %_builddir/%{name}/nvm/nvm.sh
+#install the correct node version locally
+nvm install "$(cat .node-version)"
```

Comment:
@alexweltman, @john-gress is going to talk with  everyone to get a consensus on how we approach the dependency management for kibana.